User Story 1: Task 2: Courses & Assignments Sections

HenrikLeutscher opened the pull request:
Added the components for the following:

My Courses
My Assignments

for both Mobile and Desktop view, alongside the navigation for when these pages are made.

### What does this PR do?
Adds the following components alongside the navigation for when the component is pressed:
My Courses
My Assignments

### How should this be manually tested?
Once downloaded, both student-dashboard.html aswell as global.css open in LiveServer on VSCode and test / preview from there.

HenrikLeutscher committed:
Task 2: Courses & Assignments Sections

Added the components for the following:

My Courses
My Assignments

for both Mobile and Desktop view, alongside the navigation for when these pages are made.

kevindjenhansen reviewed (changes requested):
Same as the other PR I would like for these changes
- Make sure to use the students initials.png in the img folder for when on desktop view
- Apply the correct fonts and font bold, the "my results" and "log out" should use Roberto but it looks like you've used Poppins

In addition I would like for these changes
- Position the "See Profile" and "Log Out" buttons on the 2nd row at the bottom in the middle, so it matches up with the other dashboards since you've added additional components (See the teacher dashboard on Figma for reference to what I mean), will also send a picture to reference what I mean easier
![ObtqB7E](https://github.com/user-attachments/assets/c8e3b6d9-527f-4d7f-84c9-edda4c719fcd)
- On mobile view make sure the images used for the buttons are centered

HenrikLeutscher committed:
Review Changes: User Story 1: Task 2: Courses & Assignments Sections

Changed the following:

Changed student initials.png to the correct one in the IMG folder when on desktop view.

Applied the correct fonts and font variants

Positioned the "My Result" and "Log Out" underneath the 3 on the top aswell as aligned them in the middle of these 3

Made all images centered to the text on Mobile View and Desktop View

kevindjenhansen reviewed (changes requested):
Could you swap the position of the See Profile button with the My Results so it matches more up with the design of the other dashboards?

HenrikLeutscher commented:
> Could you swap the position of the See Profile button with the My Results so it matches more up with the design of the other dashboards?

Yes of course I can, desktop only or do you wish the change on mobile view aswell?

kevindjenhansen commented:
> > Could you swap the position of the See Profile button with the My Results so it matches more up with the design of the other dashboards?
> 
> Yes of course I can, desktop only or do you wish the change on mobile view aswell?

Desktop only should be fine

HenrikLeutscher committed:
2. Review Changes: User Story 1: Task 2: Courses & Assignments Sections

Swapped positions of  "See Profile" and "My results" with each other on desktop view.

kevindjenhansen reviewed (changes requested):
Everything seems great but there's a change I'd like before we can put both of your tasks into "Done".

I would like for you to change the header with the header code in the header.html file instead, that way we are all using the same headers across pages so the design will be consistent.

HenrikLeutscher committed:
Header Update + Removing global.css

Change the header to the exact one in header.html, changed the navigation # to navigate to correct pages.

Deleted the global.css

kevindjenhansen reviewed (dismissed):
Great work!

kevindjenhansen committed:
Merge branch 'develop' into User-Story-1-Implement-the-Student-Dashboard-Task-2-Courses-&-Assignments-Sections
